src/world/Explosion.php

```diff
@@ -118,6 +130,9 @@ public function explodeA() : bool{
 							}
 
 							$state = $subChunk->getBlockStateId($vBlockX & SubChunk::COORD_MASK, $vBlockY & SubChunk::COORD_MASK, $vBlockZ & SubChunk::COORD_MASK);
+							if(count($this->excludedBlockTypeIds) > 0 && isset($this->excludedBlockTypeIds[$blockFactory->fromStateId($state)->getTypeId()])){
+								continue;
+							}
```

Comment:
I don't think this is right. It might end up making explosions more powerful in water since the blast force isn't being decreased. I think the previous check by overriding the blast resistance was more correct.